cprover-rust-api/build.rs

```diff
@@ -0,0 +1,19 @@
+use std::path::Path;
+
+fn main() {
+    let cpp_api_path = Path::new("../cprover-cpp-api/src");
+    println!("{}", cpp_api_path.display());
+    let _build = cxx_build::bridge("src/lib.rs")
+        .include(cpp_api_path)
+        .flag_if_supported("-std=c++11")
+        .compile("cprover-rust-api");
+
+    println!("cargo:rustc-link-search=native=/Users/fotiskoutoulakis/Devel/cbmc/build/lib/");
```

Comment:
Yes, yes - I'm aware. I will fix this once I rebase on top of some changes we expect to go in soon that will affect the C++ side of things, forcing some changes to this bit as well.

On the build side of thing, it will also need us to install a CMake module (`corrosion`) for building the Rust project along the C++ project, but I will do them both at the same time.